services/movie.py

```diff
@@ -0,0 +1,39 @@
+from db.models import Movie
+from db.models import models
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]

Reply:
not fixed